### PR TITLE
Replace deprecated gitbook cli with mdbook

### LINKED ISSUE #870: GitBook CLI is deprecated
https://github.com/GitbookIO/gitbook/blob/e350f4d8fc4fb0dda3f492200c4ffa1e56672143/README.md#L1-L6

ref: https://github.com/pankona/hashira/issues/869

### PR DESCRIPTION
**Pull request by kachick**
Resolves #870

If we use nix in https://github.com/pankona/hashira/issues/869,
the dependencies can be injected as `nix-shell --packages mdbook mdbook-plantuml plantuml`

### CODE REVIEW

File: docs/src/communication/overview.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Communication
+
+- Use GRPC for communication between front and daemon
+- Use GRPC for communication between daemon and datastore
```
Comment (kachick) [inline docs/src/communication/overview.md:4]: I've added this file for adjusting mdbook structure restriction.
Please update the content if you need.

**Comment by pankona** [comment]
@kachick Thank you! I will check this soon!

**Review by pankona — APPROVED** [review]
時代は mdbook であったか

**Comment by kachick** [comment]
📖 🎉

**Pull request merged**